kimchi/src/verifier.rs

```diff
@@ -511,6 +571,9 @@ where
         ));
     }
 
+    //~ Check the lengh of evaluations inside the proof.
```

Comment:
last nit! but this needs to be `//~ 1.` so that it adds to [the list here](https://o1-labs.github.io/proof-systems/specs/kimchi.html#partial-verification).